### ISSUE TITLE
SteamCleaner wants to delete more than just redistributables includes all of L4D2 Add-on Support 

### ISSUE THREAD

**Issue by jChapman** [Codeusa/SteamCleaner#45]
Maybe it knows something I don't, but why does it want to delete all of "left 4 dead 2 Add-on Support"?

![allofl4d2addonsupport](https://cloud.githubusercontent.com/assets/1152464/21559434/31a4794e-ce1a-11e6-8b55-8e4c04265a29.PNG)
![dowestillhateea](https://cloud.githubusercontent.com/assets/1152464/21559436/36ee899e-ce1a-11e6-8e69-084ab79cff46.PNG)

Then it seems that EA adds some help files to all of their games that SteamCleaner wants to delete:
![htmlfiles](https://cloud.githubusercontent.com/assets/1152464/21559438/37e5bf02-ce1a-11e6-8b24-3778b689a532.PNG)
![moreea](https://cloud.githubusercontent.com/assets/1152464/21559439/390efef2-ce1a-11e6-998b-08faed3363c5.PNG)

Eh, I don't really know what this is, but it doesn't look like reditsts to me:
![notredis](https://cloud.githubusercontent.com/assets/1152464/21559441/3e0bf9f0-ce1a-11e6-85eb-26029d347542.PNG)

**Comment by Rohaq** [comment]
Yeah, this looks a little overzealous - or maybe it's just cleaning up files that are included in Left 4 Dead's main installation now - but without any explanation of what ruleset it falls under, or why it's being removed, it's difficult to judge!

Plus there's no obvious way to whitelist anything that you don't want to risk deleting - an exclusion tickbox for rules, individual files, or for an entire directory under Steamapps/common would be welcome!

**Comment by Rohaq** [comment]
Come to think of it, it looks like this just tries to delete any path in your Steamapps directory with "Support" in it, which is why "left 4 dead 2 Add-on Support" is flagged.

Which is just crazy dangerous.
